task9/Trainings_Pages/css/Styles.css

```diff
@@ -0,0 +1,442 @@
+    padding: 0 10px;
+    line-height: 25px;
+}
+.programs_demanted{
+    min-width: 250px;
+    padding: 0 20px;
+    height: 50px;
+    float: left;
+    margin: 5px;
+    border-radius: 3px;
+    background-color: white;
+    color: #1877CC;
+    line-height: 50px;
+    text-align: center;
+    cursor: pointer;   
+}
+.programs_demanted:hover{
+    box-shadow: -10px 0px 13px -7px #645f5f, 10px 0px 13px -7px #b9aeae, 
+    inset -7px 4px 15px -3px rgba(177, 165, 165, 0);
+    text-decoration: underline;
+    transition: 1s;
+    color: rgb(12, 12, 110);
+    transform: scale(1.05);
+    border-radius: 3px;
+}
+.table{
+    color: #1877CC;
+    cursor: pointer;
+}
+.table:hover{
+    color: rgb(14, 14, 136);
+}
+.colomn{
+    display: inline-block;
+    border: 1px solid rgb(73, 211, 73);
+    background-color: rgb(33, 139, 33);
+    cursor: pointer;
+    color: white;
+    width: 150px;
+    height: 30px;
+    line-height: 30px;
+    margin: 5px;
+    border-radius: 3px;
+    text-decoration: none;
+    text-align: center;
+}
+.colomn:hover{
+    background-color: rgb(73, 211, 73);
+    box-shadow: -10px 0px 13px -7px #000000, 10px 0px 13px -7px #000000, 
+    inset -7px 4px 15px -3px rgba(83,83,83,0);
+    transition: 1s;
+}
+.colomn2{
+    display: inline-block;
+    border: 1px solid rgb(61, 160, 190);
+    background-color: #f2f2f2;
+    cursor: pointer;
+    color: #1877CC;
+    width: 150px;
+    height: 30px;
+    line-height: 30px;
+    margin: 5px;
+    border-radius: 3px;
+    text-decoration: none;
+    text-align: center;
+}
+.colomn2:hover{
+    background-color: white;
+    box-shadow: -10px 0px 13px -7px #000000, 10px 0px 13px -7px #000000, 
+    inset -7px 4px 15px -3px rgba(83,83,83,0);
+    transition: 1s;
+}
+.feedback{
+    background-color: white;
+    padding: 20px;
+    margin: 15px 8px 8px;
+    border-radius: 10px;
+}
+.feedback:hover{
+    box-shadow: -13px 14px 7px -5px rgba(34, 60, 80, 0.2);
+}
+.avatar{
+    float: left; 
+    overflow: hidden; 
+    border-radius: 100%; 
+    margin: 10px;
+    padding-right: 10px;
+}
+.avatar:hover{
+    transform: scale(1.2);
+    transition: 3s;
+}
+.card_course{
+    display: block;
+    color: #706f6f;
+    text-decoration: none;
+    padding-top: 15px;
+    max-width: 400px;
+}
+.card_course:hover{
+    color: #1877CC;
+}
+.card_name{
+    font-size: 24px;
+    font-weight: bold;
+    margin-top: 10px;
+}
+.card_footer{
+    font-size: 12px;
+    padding-top: 10px;
+}
+.card_footer_link{
+    margin: 10px;
+    color: blue;
+    text-decoration: none;
+}
+.text_feedback{
+    max-height: 100px;
+    overflow: hidden;
+    margin-top: 20px;
+    line-height: 25px;
+    text-align: justify;
+}
+.text_feedback:before{
+    content: "";
+    display: block;
+    height: 20px;
+    position: relative;
+    top: 73px;
+    background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0), #fff 75%)
+}
+.modal{
+    position: fixed;
+    font-family: Roboto;
```

Comment:
You have already set this font in the body

Reply:
Deleted this font-family out .modal